images/linux/scripts/installers/nodejs.sh

```diff
@@ -9,6 +9,16 @@ curl -sL https://raw.githubusercontent.com/mklement0/n-install/stable/bin/n-inst
 ~/n/bin/n lts
 npm install -g grunt gulp n parcel-bundler typescript newman
 npm install -g --save-dev webpack webpack-cli
+
+# Install the Netlify CLI using --unsafe-perm=true --allow-root options to avoid permission issues
+sudo npm i --unsafe-perm=true --allow-root -g netlify-cli
+
+# Install the Vercel CLI
+npm i -g vercel
```

Comment:
I think we can safely attach it to line 11 instead of having a separate installation line:
npm install -g grunt gulp n parcel-bundler typescript newman **vercel**

Reply:
Thank you, attached vercel to [line 10](https://github.com/actions/virtual-environments/pull/2666/files#diff-1192759f493cf56d17d597b2e9a82ec821f3c9548d4364f4451cbbb4636c0d1bR10)